dmscripts/helpers/auth_helpers.py

```diff
@@ -9,17 +9,21 @@ def get_auth_token(api, stage):
     if stage.lower() in DEV_ALIASES:
         return 'myToken'
 
-    DM_CREDENTIALS_REPO = os.environ.get('DM_CREDENTIALS_REPO', '../digitalmarketplace-credentials')
-    token_prefix = 'J' if subprocess.check_output(["whoami"]) == 'jenkins' else 'D'
-    creds = subprocess.check_output([
-        "{}/sops-wrapper".format(DM_CREDENTIALS_REPO),
-        "-d",
-        "{}/vars/{}.yaml".format(DM_CREDENTIALS_REPO, stage)
-    ])
-    auth_tokens = yaml.load(creds)[api]['auth_tokens']
-    auth_token = [token for token in auth_tokens if token.startswith(token_prefix)]
-
-    return auth_token[0]
+    env_var = "DM_{}_TOKEN_{}".format(api.upper(), stage.upper())
+    auth_token = os.environ.get(env_var)
+
+    if not auth_token:
+        DM_CREDENTIALS_REPO = os.environ.get('DM_CREDENTIALS_REPO', '../digitalmarketplace-credentials')
+        token_prefix = 'J' if subprocess.check_output(["whoami"]) == 'jenkins' else 'D'
+        creds = subprocess.check_output([
+            "{}/sops-wrapper".format(DM_CREDENTIALS_REPO),
+            "-d",
+            "{}/vars/{}.yaml".format(DM_CREDENTIALS_REPO, stage)
+        ])
+        auth_tokens = yaml.safe_load(creds)[api]['auth_tokens']
```

Comment:
Have updated the yaml loading, thanks @MuriloDalRi

Reply:
👍 I've added a few PRs to change the other apps as well